gateway/utils/store.go

```diff
@@ -89,6 +91,81 @@ func LoadValue(key string, state map[string]interface{}) (interface{}, error) {
 				return nil, fmt.Errorf("invalid type found for length")
 			}
 		}
+		if strings.HasPrefix(function, "now") {
+			return time.Now().UTC().Format(time.RFC3339), nil
+		}
+		if strings.HasPrefix(function, "addDuration") {
+			params0 := strings.ReplaceAll(params[0], " ", "")
+			params0 = strings.Trim(params0, "'")
+			params1 := strings.ReplaceAll(params[1], " ", "")
+			params1 = strings.Trim(params1, "'")
+
+			if strings.HasPrefix(params0, "utils") {
+				temp, err := LoadStringIfExists(params0, state)
+				if err != nil {
+					return "", err
+				}
+				params0 = temp
+			}
+
+			paresedtime, err := time.ParseDuration(params1)
+			if err != nil {
+				return "", err
+			}
+
+			param0 := time.Now().UTC()
+			param0, err = time.Parse(time.RFC3339, params0)
+			if err != nil {
+				param0, err = time.Parse("2006-01-02", params0)
+				if err != nil {
+					return nil, fmt.Errorf("invalid date format (%s) provided", params0)
+				}
+			}
```

Comment:
We are using it in a lot of places. Make a function in `utils/time.go` and use that everywhere